### PR TITLE
Documentation - Add recent third party tutorials

### PR DESCRIPTION
**Pull request by lb-**
* Add some recent (last few months) third party tutorials.

### CODE REVIEW

**Comment by lb-** [comment]
Would be good if we can backport this to 6.3 (if approved) so these are in the latest stable.

File: docs/advanced_topics/third_party_tutorials.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ from third-party developers. Some of the older links may not apply to
 the latest Wagtail versions.
 ```
 
+-   [Adding Wagtail to a Django project to support content](https://medium.com/@zalun/base-6-wagtail-cms-for-the-content-f60d8ec7ac9e) (29 September 2024)
+-   [A simple block pattern for Wagtail CMS](https://lincolnloop.com/insights/simple-block-pattern-wagtail-cms/) (29 August 2024)
+-   [Django and Wagtail site building comparison tutorial video](https://www.youtube.com/watch?v=qodE7XyWCVw) (9 July 2024)
+-   [An introduction to Wagtail tutorial](https://medium.com/@dogung51290/wagtail-cms-tutorial-5bd39e08ad44) (17 June 2024)
+    [Unleashing the Power of Custom Wagtail Models](https://www.linkedin.com/pulse/unleashing-power-custom-wagtail-models-kgothatso-phooko-70tcf) (9 June 2024)
```
Comment (thibaudcolas) [inline docs/advanced_topics/third_party_tutorials.md:13]:
```suggestion
-   [Unleashing the Power of Custom Wagtail Models](https://www.linkedin.com/pulse/unleashing-power-custom-wagtail-models-kgothatso-phooko-70tcf) (9 June 2024)
```

**Review by thibaudcolas — APPROVED** [review]
Thank you!

**Comment by lb-** [comment]
Thanks @thibaudcolas

**Pull request merged**